src/simulation/m_mpi_proxy.fpp

```diff
@@ -2333,6 +2333,13 @@ contains
 
     end subroutine s_mpi_sendrecv_capilary_variables_buffers
 
+    subroutine s_mpi_send_random_number(phi_rn)
```

Comment:
what is the purpose of this?

Reply:
This ensures the random phase shift is the same for all the ranks as the acoustic source support can span multiple ranks.

Reply:
Does this need a separate subroutine, or can you just add this to an existing MPI broadcast subroutine (wrapped in an `if(broadband_source)` statement)?